src/main/resources/META-INF/rust-core.xml

```diff
@@ -985,6 +985,13 @@
             <add-to-group group-id="RefactoringMenu"/>
         </action>
 
+        <action id="Rust.RsExtractEnumVariant"
+                class="org.rust.ide.refactoring.extractEnumVariant.RsExtractEnumVariantAction"
+                text="Extract Enum Variant"
+                description="Extract a single enum variant into a separate struct">
+            <add-to-group group-id="RefactoringMenu"/>
```

Comment:
I think it should be located in `Refactor | Extract`. `IntroduceActionsGroup` is the corresponding id

Reply:
Thank you, I wanted to put it under Extract, but I didn't know how to do it, because the only other refactoring in this group (Extract Function) is built-in via `RsRefactoringSupplier` and I didn't find it in the XML. I also added `anchor="after" relative-to-action="ExtractMethod"`, otherwise there was a needless horizontal line between those two refactorings.